packages/open-api-page/partials/collapsible/index.js

```diff
@@ -0,0 +1,91 @@
+import React, { useRef, useEffect } from 'react'
+import s from './style.module.css'
+
+/**
+ * Collapsible content block, receives children and a `isCollapsed` prop,
+ * smoothly animates from collapsed (height 0) to expanded (height auto)
+ * when its incoming `isCollapsed` prop is changed
+ * @param {*} props
+ * @param {boolean} props.isCollapsed if true, the collapsible area with occupy zero height. automatically animates between states.
+ * @param {*} props.children React children to render in the collapsible area
+ * @returns
+ */
+function Collapsible({ isCollapsed, children }) {
```

Comment:
Note: We have a task to add `Collapsible` as a `react-component` package proper, but didn't swing it this time around. [Task link](https://app.asana.com/0/1100423001970639/1157821186237305/f) for reference.